### DISCUSSION TITLE
关于最新8月7日与8月6日版本的问题

### DISCUSSION THREAD

**Discussion by tangchuqi** [shuax/MouseInc.Settings#107] [Show and tell]
开发者你好，我非常喜欢这款软件，也很信任软件。由于公司电脑强制安装了kaspersky，而且无法退出无法卸载无法添加白名单目录和文件，在8月6日之前的版本都能共存，但是新版本被误报了（撤回删除的文件几分钟又删了），所以我只能使用回旧版本。也可能是因为新特性申请了某些权限导致被卡巴斯基误报。我特地反馈一下，如果是因为无意写的BUG可以修复就修复，新版本特性的话我就用回旧版本。感谢作者的耐心阅读

**Comment by shuax** [comment]
杀毒软件问题自己处理
